package.json

```diff
@@ -1,12 +1,11 @@
 {
   "name": "express-authorshaven",
   "version": "1.0.0",
-  "type": "module",
   "description": "A Social platform for the creative at heart",
   "main": "src/index.js",
   "scripts": {
     "start": "node --experimental-json-modules src/index.js",
-    "test": "nyc --reporter=text mocha --experimental-json-modules --exit",
+    "test": "nyc --reporter=text mocha  --require @babel/register --experimental-json-modules --exit ",
     "dev": "nodemon --exec babel-node --experimental-json-modules src/index.js",
     "coverage": "nyc --reporter=lcov --reporter=text-lcov --all -x \"./node_modules/\" -x \"./coverage/\" npm run test && nyc report --reporter=text-lcov | coveralls"
```

Comment:
check in Slack a new way to wire coverage script

Reply:
> check in Slack a new way to wire coverage script

done!